doc/sphinx/proofs/writing-proofs/index.rst

```diff
@@ -10,19 +10,17 @@ the user and the assistant.  The building blocks for this dialog are
 tactics which the user will use to represent steps in the proof of a
 theorem.
 
-Incomplete proofs have one or more open (unproven) sub-goals.  Each
-goal has its own context (a set of assumptions that can be used to
-prove the goal).  Tactics can transform goals and contexts.
-Internally, the incomplete proof is represented as a partial proof
-term, with holes for the unproven sub-goals.
+A first section presents the proof mode (the core mechanism of the
+dialog between the user and the proof assistant).  Then, several
+sections describe the available tactics.  One section is dedicated to
+the SSReflect proof language, which proposes a consistent set of
+tactics to replace some of the standard basic tactics.  Finally, the
+last section documents the ``Scheme`` family of commands, which can be
+used to extend the power of the :tacn:`induction` and
+:tacn:`inversion` tactics.
```

Comment:
```suggestion
The first section presents the proof mode (the core mechanism of the
dialog between the user and the proof assistant).  Then, several
sections describe the available tactics.  One section covers
the SSReflect proof language, which proposes a consistent set of
tactics to replace some of the standard basic tactics.  The
last section documents the ``Scheme`` family of commands, which can be
used to extend the power of the :tacn:`induction` and
:tacn:`inversion` tactics.
```

For SSR, instead of "which proposes a consistent set of
tactics to replace some of the standard basic tactics", how about
"provides a consistent alternative set of tactics to the standard basic tactics"?